csrc/ops/composite.cpp

```diff
@@ -620,6 +622,40 @@ SdpfaFwdResult sdpfa_fwd(
   // value to output. Note: There is no mapping for S, E. This may change in the
   // future if we add additional reduction ids to the output.
   auto ndims_out = query_domain.size();
+  auto validate_bias_like = [&](TensorView* tv, const char* name) {
```

Comment:
`validate_bias_like` -> `validate_bias`?

What is the `like` for? The name made me think there would be another tv to compare against that is passed to the function.